PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/ViewAnnotation.java

```diff
@@ -22,7 +25,13 @@
 
   private static final double MINIMUM_LABEL_WIDTH = 10.0;
   private static final double MINIMUM_LABEL_HEIGHT = 12.0;
+  private static final double MINIMUM_ARROW_WIDTH = 25.0;
```

Comment:
If the annotation is smaller than `MINIMUM_LABEL_WIDTH` the label (text) isn't drawn. If the annotation is smaller than `MINIMUM_ARROW_WIDTH` the arrow won't be drawn.